### ISSUE TITLE
TypeError: Cannot read property 'WyvernExchange' of undefined

### ISSUE THREAD

**Issue by Benjythebee** [ProjectOpenSea/embeddable-nfts#11]
I have this other issue present with mainnet NFTs:

![image](https://user-images.githubusercontent.com/38708022/81139851-5f5fed80-8fbb-11ea-9810-71243cc85d76.png)

it infinitely loads with the following error:

![image](https://user-images.githubusercontent.com/38708022/81140135-6dfad480-8fbc-11ea-8bc9-b77ee6a8fbe8.png)

This is using [this asset](https://opensea.io/assets/0xb932a70a57673d89f4acffbe830e8ed7f75fb9e0/5536)

> 0xb932a70a57673d89f4acffbe830e8ed7f75fb9e0
token id: 5536
network: mainnet

Obviously I've tried with other mainnet nfts and it won't work.

However, rinkeby items seem to work:

![image](https://user-images.githubusercontent.com/38708022/81139879-7ef71600-8fbb-11ea-81be-418b4692ff43.png)

Here is the code I used for both cards.

```
<nft-card tokenAddress="0xb932a70a57673d89f4acffbe830e8ed7f75fb9e0"
              tokenId="5536"
              network="mainnet"
              width="15rem">
            </nft-card>
```

I have tried using both 
`<script src="https://unpkg.com/embeddable-nfts/dist/nft-card.min.js"></script>`

and
`<script src="https://unpkg.com/embeddable-nfts@0.4.6/dist/nft-card.min.js"></script>`

**Comment by taylorjdawson** [comment]
@Benjythebee there was an issue with using `mainnet` as the network name. Should be fixed now in `v0.4.7`. If you could try that and let me know if it works and I'll close this, thank you!

**Comment by Benjythebee** [comment]
Switched it to 0.4.7 and it works!

![image](https://user-images.githubusercontent.com/38708022/81228836-e310ed00-9042-11ea-88e4-a1e30ba78e9e.png)

Thanks!
